govcd/catalog.go

```diff
@@ -279,6 +296,7 @@ func (cat *Catalog) UploadOvf(ovaFileName, itemName, description string, uploadP
 // uploadPieceSize - size of chunks in which the file will be uploaded to the catalog.
```

Comment:
Please consider renaming to `uploadChunkSize` in all places

Reply:
I renamed it to piece due that we have already chunkedOVA which means that vmkds inside are splitted. And upload splitting is call pieceSize to misinterpret with chunkedOVA.

Reply:
Ah! Got it!